lib/std/Build.zig

```diff
@@ -973,6 +975,38 @@ pub fn addStaticLibrary(b: *Build, options: StaticLibraryOptions) *Step.Compile
     });
 }
 
+pub const LibraryOptions = struct {
+    linkage: std.builtin.LinkMode,
```

Comment:
```suggestion
    linkage: std.builtin.LinkMode = .static,
```

Reply:
> ```suggestion
>     linkage: std.builtin.LinkMode = .static,
> ```

Thanks, makes sense, pushed